src/Cli/dotnet/Parser.cs

```diff
@@ -269,6 +268,15 @@ public override void Write(HelpContext context)
                 {
                     new VSTestForwardingApp(helpArgs).Execute();
                 }
+                else if (command.Name.Equals(FormatCommandParser.GetCommand().Name))
+                {
+                    var argumetns = context.ParseResult.GetValueForArgument(FormatCommandParser.Arguments);
+                    new DotnetFormatForwardingApp(argumetns.Concat(helpArgs).ToArray()).Execute();
+                }
+                else if (command.Name.Equals(FsiCommandParser.GetCommand().Name))
+                {
+                    new FsiForwardingApp(helpArgs).Execute();
+                }
```

Comment:
This is great for this PR (and nothing that you;ve done incorrectly), but the more I see this the more I want to extract all of these to explicit subcommands of `System.CommandLine.Command`, so that they could override their individual help implementations/potentially do completions/etc. Making a note to myself here :)

Reply:
I agree with you. 
But don't you think that it's hidden dependency? I mean a situation when DotnetFormat team add new subcommand and SDK team should add this subcommand too for correct help/completition. 
If we could deliver DotnetFormat via git submodule/nuget I think it will solve this situation.